syntax = "proto3";

package auth;

service AuthService 
{
  rpc Login(LoginRequest) returns (TokenResponse);
  rpc Register(RegisterRequest) returns (TokenResponse);
}

message RegisterRequest {
  string email = 1;
  string password = 2;
}

message LoginRequest 
{
  string email = 1;
  string password = 2;
}

message TokenResponse 
{
  string access_token = 1;
  string token_type = 2;
}
